presto-main/src/main/java/com/facebook/presto/server/BasicQueryInfo.java

```diff
@@ -17,13 +17,14 @@
 import com.facebook.presto.execution.QueryId;
 import com.facebook.presto.execution.QueryInfo;
 import com.facebook.presto.execution.QueryState;
+import com.facebook.presto.spi.StandardErrorCode;
+import com.facebook.presto.spi.StandardErrorCode.ErrorType;
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import io.airlift.units.Duration;
 import org.joda.time.DateTime;
 
 import javax.annotation.concurrent.Immutable;
-
```

Comment:
This blank line should stay per our code style: https://github.com/airlift/codestyle